settings.gradle

```diff
@@ -8,5 +8,4 @@
  */
 
 rootProject.name = 'varadhi'
-include('entities', 'common', 'core', 'messaging', 'pulsar', 'server', 'authz', 'consumer')
-
+include('entities', 'spi', 'common', 'core', 'messaging', 'pulsar', 'server', 'authz', 'consumer')
```

Comment:
library.

Reply:
ok, let's see in that case if consumer can be wrapped under messaging library module itself along with producer.
This will be independent of this PR though.